ambersim/utils/io_utils.py

```diff
@@ -6,27 +6,33 @@
 import numpy as np
 import trimesh
 from dm_control import mjcf
+from dm_control.mjcf.element import _ElementImpl
+from dm_control.mjcf.parser import from_file
+from dm_control.mjcf.physics import Physics
 from mujoco import mjx
+from mujoco.mjx._src.device import device_put
+from mujoco.mjx._src.io import make_data
 
 from ambersim import ROOT
-from ambersim.utils._internal_utils import _check_filepath
+from ambersim.utils._internal_utils import check_filepath
 from ambersim.utils.conversion_utils import save_model_xml
 
 
-def _modify_robot_float_base(filepath: Union[str, Path]) -> mj.MjModel:
+def _modify_robot_float_base(filepath: str) -> mj.MjModel:
```

Comment:
Example: I just copy/pasted this type hint from other function signatures, but `pyright` complains below that objects of type `Path` don't have a `split` function. It turns out that I only ever pass in `str` args into this function, so I changed the type hint to reflect that.